pages/claims/info.js

```diff
@@ -1,52 +1,60 @@
-
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { bindActionCreators } from 'redux';
 import withRedux from 'next-redux-wrapper';
 import configureStore from '../../store/configureStore';
-import Layout from '../../layouts/Main';
-import showClaims from '../../actions/showClaims';
-
-const showClaimPage = (props) => {
-  const { claims } = props;
-  const infoArray = claims[0].text.split(',');
-
-  return (
-    <Layout>
-      <h2>Claim full content:</h2>
-      <a>Part A: {infoArray[0]}</a><br />
-      <a>Part B: {infoArray[1]}</a><br />
-      <a>Expiration Date: {infoArray[2]}</a><br />
-      <a>Image: </a><br />
-      <img src={infoArray[3]} alt="" /><br />
-      <a>Description: {infoArray[4]}</a>
-    </Layout>
-  );
-};
-
-showClaimPage.propTypes = {
-  claims: PropTypes.array,
+import Layout from '../../layouts/material/Main';
+
+import { fetchClaimInfo } from '../../utils/tangleidAPI';
```

Comment:
It might be annoying to specify `tangleidAPI`. Can it be feasible to get reduced into `API` or something alike?

Reply:
I think this is necessary to specify `tangleidAPI` if we use more than one different APIs in our application.
Moreover, I'm planning to move the `tangleidAPI` to an NPM package, then everyone can use TangleID API. 😃

Reply:
TangleID is conceptually an interface to bridge IOTA transactions and identity along with advanced features such MAM and extension-specific filters. The APIs provided by TangleID should be minimized as possible as we can, and the use cases like LASS or Taipei City Card would be no doubly built as extensions to TangleID. The more APIs exposed, the harder we can maintain in elegant ways.

Reply:
Maybe we could take `iota.lib.js` as a [reference](https://github.com/iotaledger/iota.lib.js/#getting-started). 

Change the API as the following example. 🤔 
```
// Setup the TangleID instance
import TangleID from 'TangleID';
var tangleid = new TangleID({
    'provider': SWARM_NODE_ADDRESS
});

// Call TangleID API
tangleid.api.fetchUserList();

// Call extension APIs
tangleid.extenstion.method();
```

Reply:
I'll open a new issue to describe this plan.

Reply:
Let's ensure the consistency of API scheme first and then merge this pull request.

Reply:
Roger that. 👍

Reply:
This pull request depends on #54